tests/sieves/backports/test_enum34.py

```diff
@@ -92,49 +87,21 @@ def test_no_include(
         builder_context.recommendation_type = recommendation_type
         builder_context.decision_type = decision_type
 
-        package_version = PackageVersion(
-            name="enum34", version="==1.1.10", develop=develop, index=Source("https://pypi.org/simple"),
-        )
-
-        builder_context.project.pipfile.add_package_version(package_version)
-
         assert builder_context.is_dependency_monkey_pipeline() or builder_context.is_adviser_pipeline()
-        assert Enum34BackportBoot.should_include(builder_context) is None
+        assert Enum34BackportSieve.should_include(builder_context) is None
 
     def test_remove(self, context: Context) -> None:
         """Test removing enum34 dependency."""
         package_version = PackageVersion(
             name="enum34", version="==1.1.10", develop=False, index=Source("https://pypi.org/simple"),
         )
 
-        context.project.pipfile.add_package_version(package_version)
-        assert "enum34" in context.project.pipfile.packages.packages
-        assert "enum34" not in context.project.pipfile.dev_packages.packages
-        assert not context.stack_info
-
-        with Enum34BackportBoot.assigned_context(context):
-            unit = Enum34BackportBoot()
-            assert unit.run() is None
-
-        assert len(context.stack_info) == 1
-        assert "enum34" not in context.project.pipfile.packages.packages
-        assert "enum34" not in context.project.pipfile.dev_packages.packages
-
-    def test_remove_develop(self, context: Context) -> None:
-        """Test removing develop enum34 dependency."""
-        package_version = PackageVersion(
-            name="enum34", version="==1.1.10", develop=True, index=Source("https://pypi.org/simple"),
-        )
-
-        context.project.pipfile.add_package_version(package_version)
-        assert "enum34" not in context.project.pipfile.packages.packages
-        assert "enum34" in context.project.pipfile.dev_packages.packages
-        assert not context.stack_info
+        unit = Enum34BackportSieve()
+        with Enum34BackportSieve.assigned_context(context):
+            with pytest.raises(SkipPackage):
+                list(unit.run(pv for pv in (package_version,)))
 
-        with Enum34BackportBoot.assigned_context(context):
-            unit = Enum34BackportBoot()
-            assert unit.run() is None
+            with pytest.raises(SkipPackage):
```

Comment:
Is this duplicate necessary?

Reply:
I was checking the `stack_info` has just one message even though the unit is called multiple times. See the assert below. I can add a comment if you want.

Reply:
I see. I think a comment would be nice here +1

Reply:
Added PTAL